specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2021-09-01-preview/appplatform.json

```diff
@@ -3462,6 +3462,14 @@
           "description": "Indicate if end to end TLS is enabled.",
           "type": "boolean",
           "default": false
+        },
+        "loadedCertificates": {
+          "description": "Collection of loaded certificates",
+          "uniqueItems": false,
```

Comment:
Not quite understand this property. Whether it means this array can contain dup item?